README.md

```diff
@@ -8,14 +8,28 @@ Contributors and beta testers are welcome to go through the course, raise issues
 
 Contact Brooklin (brooklin.myers@dockyard.com) or DM at [@BrooklinJMyers](https://twitter.com/BrooklinJMyers) on Twitter if you would like more information.
 
-# QuickStart
-1. install a compatible Elixir and Erlang version. You may wish to use [asdf](https://asdf-vm.com/guide/getting-started.html#_1-install-dependencies).
-2. Install livebook main `mix escript.install github livebook-dev/livebook`.
-3. From the project folder run `livebook server start.livemd`. This opens the navigation page where you can find the course reading material and associated exercises.
+## QuickStart
 
-If you cannot see mermaid.js graphs, ensure your livebook version is correct.
+1. Install a compatible Elixir (1.13.3 or higher) and Erlang (24 or higher)
+   version. You may wish to use
+   [asdf](https://asdf-vm.com/guide/getting-started.html#_1-install-dependencies).
 
-See the full [Beta Tester & Contributor Setup](https://github.com/DockYard-Academy/beta_curriculum/wiki/Beta-Tester-Guide-&-Contributor-Setup) for comprehensive getting started and contribution instructions.
+2. Install [Livebook](https://github.com/livebook-dev/livebook):
+
+   ```sh
+   mix escript.install github livebook-dev/livebook
+   ```
+
+3. From the project folder run the command below. This opens the navigation
+   page where you can find the course reading material and associated
+   exercises.
+
+   ```sh
+   livebook server start.livemd
+   ```
+
+If you cannot see [mermaid.js](https://github.com/mermaid-js/mermaid) graphs,
+ensure your Livebook version is correct.
 
```

Comment:
Where can we see any Mermaid graph?

Reply:
I believe the first mermaid graph used in the course is in the arithmetic section.

It could be useful to have a test for it. Maybe we can include a sample mermaid graph in start.livemd🤔